src/call_builder.js

```diff
@@ -30,6 +30,12 @@ export class CallBuilder {
     }
 
     if (this.filter.length === 1) {
+      this.filter[0].forEach(element => {
```

Comment:
Why is this only checking the first filter? Shouldn't it loop through and check them all?

Reply:
got a point. let me review.

Reply:
Oh actually I saw yesterday that calls won’t build if they have more than one filter. Not sure why we chose an array of that’s the case. So please disregard!

Reply:
Have to come back and see why I did that, there is a check on line 28 that prevents it from having more than one, and the second check on line 32 will assure I always have one element.